src/operations/CoverageMapGeneration.cpp

```diff
@@ -28,7 +28,7 @@ void CoverageTracker::updateMLS(const grid::MLSMapKalman& mls_) {
     }
 }
 
-void CoverageTracker::addCoverage(const double &radius, const base::AngleSegment& range /* ignored */, const base::Pose& pose_in_map)
```

Comment:
For easier transition we could make this a deprecated method
`addCoverage(const double &radius, decltype(std::ignore), const Eigen::Affine3d& pose_in_map)`.

Reply:
And the actual method could actually just take a position.